v23.1/connect-to-a-changefeed-kafka-sink-with-oauth-using-okta.md

```diff
@@ -0,0 +1,291 @@
+---
+title: Connect to a Changefeed Kafka sink with OAuth Using Okta
+summary: Learn how to set up OAuth through Okta for a changefeed streaming to Kafka.
+toc: true
+docs_area: stream_data
+---
+
+{% include_cached new-in.html version="v23.1" %} CockroachDB {{ site.data.products.enterprise }} changefeeds can stream change data out to [Apache Kafka](https://kafka.apache.org/) using OAuth authentication.
+
+{% include {{ page.version.version }}/cdc/oauth-description.md %}
+
+In this tutorial, you will set up OAuth authentication for your Kafka changefeed sink using [Okta](https://www.okta.com/). You will create an OAuth 2.0 endpoint in Okta that will provide a temporary credential token that the changefeed will use to connect to your Kafka cluster.
+
+An overview of the workflow involves:
+
+1. Creating an Okta application integration and authorization server scope.
+1. Configuring the Kafka cluster.
+1. Streaming the changefeed to the Kafka cluster.
+
+## Before you begin 
+
+You will need the following set up before starting this tutorial:
+
+- A CockroachDB cluster. You can use a {{ site.data.products.db }} or {{ site.data.products.core }} cluster. If you are using {{ site.data.products.serverless }} or {{ site.data.products.dedicated }}, see the [Quickstart with CockroachDB](../cockroachcloud/quickstart.html) guide. For {{ site.data.products.core }} clusters, see the [install](install-cockroachdb-mac.html) page. 
```

Comment:
```suggestion
- A CockroachDB cluster. You can use a {{ site.data.products.db }} or {{ site.data.products.core }} cluster.
    - If you are using {{ site.data.products.db }} ({{ site.data.products.serverless }} or {{ site.data.products.dedicated }}), see the [Quickstart with CockroachDB](../cockroachcloud/quickstart.html) guide.
    - For {{ site.data.products.core }} clusters, see the [install](install-cockroachdb-mac.html) page. 
```

Forgetting our bullet indentation. Please confirm.

Reply:
Did it!